src/services/stag/ExamService.js

```diff
@@ -0,0 +1,98 @@
+const BaseService = require('./BaseService');
+
+class ExamService extends BaseService{
+
+    /**
+     * @param {StagAdapter} stagAdapter
+     */
+    constructor(stagAdapter) {
+        super(stagAdapter);
+        this.serviceEndpoint = 'terminy';
+    }
+
+    /**
+     * Vrací zkouškové termíny pro přihlášeného studenta
+     * Z mandatory bloku je třeba vybrat alespoň jeden prvek pro filtrovani
+     *
+     * @param {Object}  [mandatory]
```

Comment:
[mandatory] je kontradikce - hranaté závorky říkají doslova optional

```suggestion
     * @param {Object}  filterBy - contains at least one of available properties
```